### DISCUSSION TITLE
Using git-tags or git-ref datasources in monorepos

### DISCUSSION THREAD

**Discussion by alexg-axis** [renovatebot/renovate#14520] [Request Help] [answered]
In a monorepo with tags such as `applications/fancy-app/v1.0.0`, `libraries/core-lib/v0.1.5`, the `git-tags` datasource does not currently seem as a good fit. It is of my understanding that the expected use case for `git-tags` are repositories containing a single project with semver tags such as `1.0.0`.

Is there any feasible way to make `git-tags` or `git-refs` work with monorepos?

The full use case I have is to use this in combination with the regex manager to allow renovate to keep such tags in sync with newer versions.

**Comment by rarkins** [comment, marked as answer]
Looks like same request as #14501 by @ekristen, although the tag formats in your respective examples are different. As proposed there, we need a (single) feature request with at least one reproduction so that we can figure out the best way to support.

**Reply by alexg-axis** [reply]
I've opened a feature request here: https://github.com/renovatebot/renovate/issues/14546
